apps/adminer.sh

```diff
@@ -7,8 +7,6 @@ true
 # shellcheck source=lib.sh
 . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
 
-print_text_in_color "$ICyan" "Installing and securing Adminer..."
```

Comment:
There's a reason why this comes first in all scrips. If you run it on a slow connection then it will take time to load the lib.sh and the user might think the setup-script has stopped.

Please test this branch by using the setup script and choose all apps. Then you'll see what I mean. Even with a fast connection there's a lag of about 3 seconds. If we can tell the user what's going on before he/she starts to worry, that's a good thing.

Reply:
Sounds reasonable. But I will change the lines to e.g.: "Downloading and executing Collabora.sh'

Reply:
this is fixed.